test-data/unit/check-generics.test

```diff
@@ -3124,8 +3124,8 @@ def dec(f: Callable[P, T]) -> Callable[P, List[T]]: ...
 def id(x: U) -> U: ...
 def either(x: U, y: U) -> U: ...
 def pair(x: U, y: V) -> Tuple[U, V]: ...
-reveal_type(dec(id))  # N: Revealed type is "def [T] (x: T`2) -> builtins.list[T`2]"
-reveal_type(dec(either))  # N: Revealed type is "def [T] (x: T`4, y: T`4) -> builtins.list[T`4]"
+reveal_type(dec(id))  # N: Revealed type is "def [T] (x: T`3) -> builtins.list[T`3]"
```

Comment:
Do you know why this changed?

Reply:
I have no idea to be honest. They changed in the original PR by sobolevn and I didn't know why. I'll do some more digging.

Reply:
Thanks. Not too important if it's hard to figure out, I assume these numbers can change based on all kinds of factors.